implementations.md

```diff
@@ -0,0 +1,20 @@
+# OCI Image Implementations
+
+Projects or Companies currently adopting the OCI Image Specification
+
+* [projectatomic/skopeo](https://github.com/projectatomic/skopeo)
+* [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-manifest-formats.html) ([announcement](https://aws.amazon.com/about-aws/whats-new/2017/01/amazon-ecr-supports-docker-image-manifest-v2-schema-2/))
+* [openSUSE/umoci](https://github.com/openSUSE/umoci)
+* [cloudfoundry/grootfs](https://github.com/cloudfoundry/grootfs) ([source](https://github.com/cloudfoundry/grootfs/blob/c3da26e1e463b51be1add289032f3dca6698b335/fetcher/remote/docker_src.go))
+* [Mesos plans](https://issues.apache.org/jira/browse/MESOS-5011) ([design doc](https://docs.google.com/document/d/1Pus7D-inIBoLSIPyu3rl_apxvUhtp3rp0_b0Ttr2Xww/edit#heading=h.hrvk2wboog4p))
+* [Docker](https://github.com/docker)
```

Comment:
oh right. I knew the that image-spec support was on the horizon, but didn't think it was a PR yet. Should've looked